src/tests/utils/ApiTestSteps.ts

```diff
@@ -30,116 +30,147 @@ const xmlParser = new XMLParser();
 const ajv = new Ajv({ strictTuples: false });
 
 export async function startStubServiceAndReturnSessionId(): Promise<any> {
-	const stubResponse = await stubStartPost();
+	const stubResponse = await stubStartPostByType("FACE_TO_FACE");
 	return sessionPost(stubResponse.data.clientId, stubResponse.data.request);
 }
 
-export async function stubStartPost():Promise<any> {
+export async function startStubServiceAndReturnSessionIdByType(journeyType: string): Promise<any> {
```

Comment:
Do we really think we need this abstraction? It looks like the function is only being used once

Reply:
it also looks basically the same as `startStubServiceAndReturnSessionId` ?

Reply:
The idea was to let dev's choose what journey they want, but as almost every test uses the original function we wanted to keep it as a 'default'. I will remove the original function and change all mentions of it to use the new function with the parameter though